### ISSUE TITLE
No more dev for the moment

### ISSUE THREAD

**Issue by mugnier102** [home-assistant/home-assistant.github.io#21522]
### Feedback

Should be interresting to note that this integration is not working anymore on documentation. That can change but for the moment, user can't use it for the moment.

### URL

https://www.home-assistant.io/integrations/enocean/

### Version

2022.2.2

**Comment by bdurrer** [comment]
@mugnier102 Can you please elaborate: _What_ is not working? The documentation website? The integration?
I can open the link above just fine

**Comment by mugnier102** [comment]
I did try integration. It didn't work. I may have miss a reboot but googling indicate that this module is not maintened anymore has the refactoring to be done is too huge for actual contributor. Forum indicate that a working solution is to use EnOceanToMQTT and MQTT to HA. Don't known if it still true (I tried to check change log but didn't see announce on this) but quiet dissapointting for me.
After posting my comment, I did reboot and my USB hardware start to give me some information for Temperature but no information for switch...
I must add that ha is running in docker as privilete user (so no problem for USB normally).
I did try listening the dev interface from docker exec: running fine. I didn't find any information useful informaiton in ha log explaining that docker receive well but HA didn't show informaiton.
